index.html

```diff
@@ -0,0 +1,116 @@
+  #nav-bar{
+    display: grid;
+  }
+}
+header{
+  display: flex;
+}
+</style>
+
+    <header id="header" media="screen">
+      <div class="logo">
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png">
+    </div>
+      <nav id="nav-bar">
+        <ul>
+           <li><a class="nav-link" href="#features">Features</a></li>
+           <li><a class="nav-link" href="#how it works">How it works</a></li>
+           <li><a class="nav-link" href="#pricing">Pricing</a></li>
+        </ul>
+      </nav>
+    </header>
+    <div class="info">
+       <h2>Handcrafted, home-made masterpieces</h2>
+       <form id="form" media="screen" action="https://www.freecodecamp.com/email-submit">
+         <input name="email" id="email" type="email" placeholder="Enter your email address" required>
+        <br> <input calss="getstarted" id="submit" type="submit" value="Get Started">
+       </form>
+    </div>
+    <div id="features" class="content">
+      <div>
+          <div>
+            <h2>Premium Materials</h2>
+            <p>
+              Our trombones use the shiniest brass which is sourced locally.
+              This will increase the longevity of your purchase.
+            </p>
+          </div>
+          <div>
+            <h2>Fast Shipping</h2>
+            <p>We make sure you recieve your trombone as soon as we
+               have finished making it.We also provide
+               free returns if you are not satisfied.</p>
+          </div>
+          <h2>Quality Assurance</h2>
+          <p>For every purchase you make, we will ensure
+             there are no damages or faults and we will check and
+              test the pitch of your instrument.</p>
+        </div>
+    </div>
+    <div id="how it works">
+       <iframe id="video"
+                  src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&amp;controls=0&amp;showinfo=0">
+       </iframe>
+    </div>
+    <div class="pricingbox">
+    <div class="tenor" id="pricing">
+      <h2>TENOR TROMBONE</h2>
+      <h2>$600</h2>
+      <ol>
+        <li>Lorem ipsum.</li>
+        <li>Lorem ipsum.</li>
+        <li>Lorem ipsum dolor.</li>
+        <li>Lorem ipsum.</li>
+      </ol>
+      <button type="button" value="Select">Select</button>
+    </div>
+    <div id="pricing">
+      <h2>BASS TROMBONE</h2>
+      <h2>$900</h2>
+      <ol>
+        <li>Lorem ipsum.</li>
+        <li>Lorem ipsum.</li>
+        <li>Lorem ipsum dolor.</li>
+        <li>Lorem ipsum.</li>
+      </ol>
+      <button type="button" value="Select">Select</button>
+    </div>
+    <div class="valve" id="pricing">
+      <h2>VALVE TROMBONE</h2>
+      <h2>$1200</h2>
+      <ol>
+        <li>Plays similar to a Trumpet</li>
+        <li>Great for Jazz Bands</li>
+        <li>Lorem ipsum dolor.</li>
+        <li>Lorem ipsum.</li>
+      </ol>
+      <button type="button" value="Select">Select</button>
+    </div>
+    </div>
```

Comment:
For best practice and semantics, kindly use HTML semantics like section tag instead of the div tag you used in your code to define a new section in your codebase.

Reply:
- [Optional] Using emojis adds a human touch to the review process. You can add a few from time to time. they convey emotions better. 😃